### PR TITLE
Edge: Inherit dark theme from Eclipse

### LINKED ISSUE #1529: Edge: Inherit dark theme from Eclipse
Currently the WebView2 instances use COREWEBVIEW2_PREFERRED_COLOR_AUTO, i.e. they inherit the dark/light setting from the operating system, not the Eclipse application.

E.g., when setting the preferred app theme in Windows to 'light', but Eclipse to 'dark', the Edge browser will use 'light'.

It should however inherit from Eclipse and also use 'dark'.

To do this, the API https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2profile#put_preferredcolorscheme needs to be implemented.

### PR DESCRIPTION
**Pull request by sratz**
Set ICoreWebView2Profile#put_PreferredColorScheme() based on the Eclipse theme, if this is dark.

The same way it is done in

  Display.setData(USE_DARKMODE_EXPLORER_THEME_KEY, ...)

via

  OS.SetPreferredAppMode(PreferredAppMode_...);

Fixes #1529

### CODE REVIEW

**Comment by sratz** [comment]
I plan to merge this for M2.

@HeikoKlare @amartya4256 any objections?

**Review by BeckerWdf — APPROVED** [review]
I reviewed what I think that I understand. At least in this part I did not find any issues.

File: bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Display.java [hunk 1]
```diff
@@ -270,6 +270,17 @@ public class Display extends Device implements Executor {
 	 */
 	static final String USE_DARKTHEME_TEXT_ICONS = "org.eclipse.swt.internal.win32.Text.useDarkThemeIcons"; //$NON-NLS-1$
 	boolean textUseDarkthemeIcons = false;
+	/**
+	 * Use dark prefered color scheme in Edge browser.
+	 * Note:<br>
+	 * <ul>
+	 *   <li>When setting this property on the display, it is first AND'ed with !disableCustomThemeTweaks first.
```
Comment (HeikoKlare) [inline bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Display.java:277]: Double "first"
```suggestion
	 *   <li>When setting this property on the display, it is first AND'ed with !disableCustomThemeTweaks.
```

**Review by HeikoKlare — APPROVED** [review]
The code is sound and works as expected.

**Pull request merged**